src/github.com/couchbase/sync_gateway/db/index_changes.go

```diff
@@ -11,12 +11,15 @@ package db
 
 import (
 	"fmt"
-	"time"
 
 	"github.com/couchbase/sync_gateway/base"
 	"github.com/couchbase/sync_gateway/channels"
 )
 
+const (
+	kContinuousHashFrequency = 100 // For a continuous feed, how frequently a hash value is calculated
```

Comment:
It's actually number of docs, not time - probably evidence that I need a clearer description.  I'm anticipating that this gets exposed as a config option in 1.3 (with additional documentation about tradeoffs), but doesn't hurt to include that in the code now - I'll add.